### ISSUE TITLE
[HELP] How do I proxy a single location?

### ISSUE THREAD

**Issue by Romaleks360** [tobychui/zoraxy#179]
I want to route a mydomain.com/ (root location) to a separate destination (landing page builder platform) while keeping all other locations (like mydomain.com/api, /app, etc) proxied with the regular HTTP Proxy setup

I've tried this using Redirections tab, but 2 issues occurred:
1. the filter "mydomain.com/" is selecting all locations, so my /api is also redirected. I only want to redirect root
2. redirect shows the destination domain (myproject.landing-page-builder.com) in the user address bar, while I would like to keep my original redirection URL (mydomain.com)

Redirections are obviously the wrong tool for the job, but I didn't find anything better.

This kind of setup was possible with Nginx Proxy Manager (Proxy -> Custom Locations tab), but became broken recently. Can I do the same using Zoraxy?

**Comment by tobychui** [comment]
Hmmm do you mean "Virtual Directory"?
Can you give me a list of subpaths and their proxy endpoint you want? I will see what I can help and suggest. e.g.
```
/  ->  myproject.landing-page-builder.com
/api -> localhost:8080/something/api
/res -> 192.168.0.100:80/resources
```

**Comment by Romaleks360** [comment]
That's what i meant:
```
/  ->  myproject.landing-page-builder.com (user sees mydomain.com)
/literally-any-subpath -> localhost:2048/literally-any-subpath (user sees mydomain.com/literally-any-subpath)
```
Would be nice to have this feature.

But I already solved this without Zoraxy. I moved my app and API to the subdomain (app.mydomain.com) and routed it to my server with Cloudflare DNS A-record, while the domain.com routes to the landing page. 
So no more complex setup is needed at my reverse proxy for now. This required changing the app setup, links and architecture a bit, but it seems like the more correct way to do such things. Anyway, thanks for the help.
